pages/services/kafka/2.6.0-2.2.1/started/index.md

```diff
@@ -0,0 +1,617 @@
+There are four types of installation methods:
+1. Default installation is the basic installation method. This method is used to install Kafka on a DC/OS cluster using CLI commands.
+2. Minimal installation is used to install Kafka on a local C/OS cluster using [dcos-vagrant](https://github.com/mesosphere/dcos-vagrant) and is specific for development purposes.
+3. Custom installation is used to install Kafka on DC/OS cluster using a customized JSON file.
+4. Multiple Kafka cluster installation is used to install multiple Kafka clusters using custom configurations.
+
+## Default Installation
+
+To start a basic test cluster with three brokers, run the following command on the DC/OS CLI. Enterprise DC/OS users must follow additional instructions. [More information about installing Kafka on Enterprise DC/OS](#install-enterprise).
+
+```bash
+dcos package install kafka
+```
+
+This command creates a new Kafka cluster with the default name `kafka`. Two clusters cannot share the same name, so installing additional clusters beyond the default cluster requires [customizing the `name` at install time][4] for each additional instance.
+
+All `dcos kafka` CLI commands have a `--name` argument allowing the user to specify which Kafka instance to query. If you do not specify a service name, the CLI assumes the default value, `kafka`. The default value for `--name` can be customized via the DC/OS CLI configuration:
+
+```bash
+dcos kafka --name=<kafka-dev> <cmd>
+```
+
+<p class="message--note"><strong>NOTE: </strong>Alternatively, you can <a href="https://docs.mesosphere.com/1.9/deploying-services/install/">install Kafka from the DC/OS web interface</a>. If you install Kafka from the web interface, you must install the Kafka DC/OS CLI subcommands separately. </p>
+
+Enter the following command from the DC/OS CLI:
+
+```bash
+dcos package install kafka --cli
+```
+
+## Minimal Installation
+
+For development purposes, you can use [dcos-vagrant][5] to install Kafka on a local DC/OS cluster.
+
+To start a minimal cluster with a single broker, create a JSON options file named `sample-kafka-minimal.json`:
+
+```json
+{
+    "brokers": {
+        "count": 1,
+        "mem": 512,
+        "disk": 1000
+    }
+}
+```
+
+The command below creates a cluster using `sample-kafka-minimal.json`:
+
+```bash
+dcos package install --options=sample-kafka-minimal.json kafka
+```
+
+
+## Custom Installation
+
+Customize the defaults by creating a JSON file. Then, pass it to `dcos package install` using the `--options` parameter.
+
+Sample JSON options file named `sample-kafka-custom.json`:
+
+```json
+{
+    "service": {
+        "name": "sample-kafka-custom",
+        "placement_strategy": "NODE"
+    },
+    "brokers": {
+        "count": 10,
+        "kill_grace_period": 30
+    },
+    "kafka": {
+        "delete_topic_enable": true,
+        "log_retention_hours": 128
+    }
+}
+```
+
+The command below creates a cluster using `sample-kafka.json`:
```

Comment:
Rephrase line #102: The following command creates a cluster using `sample-kafka.json`: